### DISCUSSION TITLE
Ezra Bible App 1.14

### DISCUSSION THREAD

**Discussion by tobias-klein** [ezra-bible-app/ezra-bible-app#1063] [General]
**Ezra Bible App 1.14** contains various bug fixes and enhancements.

Note-worthy enhancements and fixes:

* Introduce option for configuring typeface/font of the Bible text. ([#1007][i1007])
* Add button for copying individual commentary sections to the clipboard. ([#1041][i1041])
* Bug fix: Blocking JavaScript errors after turning off error reporting option. ([#1048][i1048])
* Bug fix: Bible commentaries and dictionaries do not render paragraphs correctly. ([#1016][i1016])
* Bug fix: Showing search results in a popup generates an empty new tab if triggered via Dictionary Panel strongs search. ([#1056][i1056])

A detailed listing of all changes can be found in the [changelog](https://github.com/ezra-bible-app/ezra-bible-app/blob/master/CHANGELOG.md).

Installation instructions can be found [here](https://ezrabibleapp.net/installation).

[i1007]: https://github.com/ezra-bible-app/ezra-bible-app/issues/1007
[i1016]: https://github.com/ezra-bible-app/ezra-bible-app/issues/1016
[i1041]: https://github.com/ezra-bible-app/ezra-bible-app/issues/1041
[i1048]: https://github.com/ezra-bible-app/ezra-bible-app/issues/1048
[i1056]: https://github.com/ezra-bible-app/ezra-bible-app/issues/1056

<hr /><em>This discussion was created from the release <a href='https://github.com/ezra-bible-app/ezra-bible-app/releases/tag/1.14.0'>Ezra Bible App 1.14</a>.</em>

**Comment by Mc-beat** [comment]
I read that it is 32bit but it should still work. hmm
.see the error.
<img width="418" alt="Ezra-Win1064bit" src="https://github.com/ezra-bible-app/ezra-bible-app/assets/104306649/2fee96a5-0847-467f-b418-83419ac57ed0">
Otherwise, the app on the Mac is really great. thank you for your work.
I would like a direct search for the verses. see screenshot from esword. This makes it a lot easier to quickly find a verse without having to open your mask and look for books. see
or even directly in the existing search field, like at Bibleserver.com
<img width="86" alt="Suchmaske" src="https://github.com/ezra-bible-app/ezra-bible-app/assets/104306649/6c76de26-1f46-4c4e-aa12-dc71f0f80a27">



<img width="498" alt="Suchfeld" src="https://github.com/ezra-bible-app/ezra-bible-app/assets/104306649/0d2fb666-b35e-48ff-a453-eb17e07f6b03">

**Reply by tobias-klein** [reply]
Hi @Mc-beat,
Thanks for your feedback. Which Windows version did you test with? 
Could you do me a favor and check whether the same error occurs with 1.13 on your Windows machine? 

The feature request about the direct lookup of Bible passages is something I can consider for a future release!

**Reply by Mc-beat** [reply]
Ok. I did all the updates and installed version 13 and then 14 and it runs great.

**Reply by tobias-klein** [reply]
I'll have a look at the feature request of direct lookup of Bible passages. I created an issue for this some time ago (see #701) and I have added it to the todo list of 1.15 just now. It is depending on the effort whether this will make it into the next release, though.

**Comment by Mc-beat** [comment]
That sounds good. Can you make a notebook? 

- That way you can create a kind of book, preferably if you can insert the verses that can be clicked on.
and 
- copy out fragments of text and add them?
You can't pass on a part of a verse.

this is solved quite well with esword

**Reply by tobias-klein** [reply]
I may have a look at esword to understand better what this functionality may look like!

Something I was thinking of in the past was something like a "Wiki" system integrated in Ezra Bible App. That would be a lot of work, though.
